modules/admin-ui-frontend/app/scripts/shared/partials/modals/embedding-code.html

```diff
@@ -0,0 +1,85 @@
+<section ng-show="open" ng-keyup="keyUp($event)" tabindex="1" class="modal modal-animation ng-hide" id="embedding-code">
+  <header>
+    <a class="fa fa-times close-modal" ng-click="close()"></a>
+    <h2 translate="CONFIRMATIONS.ACTIONS.SHOW.EMBEDDING_CODE"><!-- Confirm --></h2>
+  </header>
+
+  <script>
+    var engageUrl = "<SERVER_URL>";
+
+    $(document).ready(function() {
+      var reply = $.ajax({
+        url: "/api/info/organization/properties/engageuiurl",
+        timeout: 3000,
+        success: function(result) {
+          engageUrl = result['org.opencastproject.engage.ui.url'];
+          if (typeof engageUrl === "undefined") {
+            engageUrl = "<SERVER_URL>";
+          }
+        }
+      });
+    });
+
+    var embs = $(".embedSizeButton");
+    for (var i = 0; i < embs.length; i = i + 1) {
+      var e = embs[i];
+      e.onclick = function(event) {
+        var value = event.target ? event.target.textContent : event.toElement.textContent;
+        if (value) {
+          var embedSizeButtons = $(".embedSizeButton");
+          for (var j = 0; j < embedSizeButtons.length; j++) {
+            if (embedSizeButtons[j].id == value) {
+              embedSizeButtons[j].classList.add("embedSizeButtonSelected");
+            } else {
+              embedSizeButtons[j].classList.remove("embedSizeButtonSelected");
+            }
+          }
+          var id = $("#id_video").html();
+          var url = engageUrl + "/play/" + id;
+          var size = value.trim().split("x");
+          $("#social_embed-textarea")[0].value = '<iframe allowfullscreen src="' + url + '" style="border:0px #FFFFFF none;" name="Paella Player" scrolling="no" frameborder="0" marginheight="0px" marginwidth="0px" width="' + size[0] + '" height="' + size[1] + '"></iframe>';
```

Comment:
@gregorydlogan
Thank you for the review. The name attribute is now only **Player** instead of **Paella Player.** You mean that? The url doesn't change, right?

Reply:
Correct, `/play`/ is a magic endpoint which does the redirection to the correct player.